vue-frontend/src/components/home-new/UserReview.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="sm:tw-container tw-relative tw-pt-[50px] tw-pb-20 md:tw-pt-[100px] md:tw-pb-[150px] lg:tw-pt-[150px] lg:tw-pb-[250px]"
+    class="sm:tw-container tw-relative tw-pt-[50px] tw-pb-[5rem] sm:tw-pb-20 md:tw-pt-[100px] md:tw-pb-[150px] lg:tw-pt-[150px] lg:tw-pb-[250px]"
```

Comment:
Isn't it work by just making `pb` from `sm`?

Reply:
no, it wasn't working that's why need to use sm.